tests/unit/Pipeline.cpp

```diff
@@ -146,6 +147,12 @@ class MapContainer : public Container<MapContainer> {
     return true;
   }
 
+  llvm::Error
+  extractOne(llvm::raw_ostream &OS, const Target &Target) const override {
+    revng_check(false);
```

Comment:
`revng_abort`. Not the first time, please `git grep revng_check(false)` and make sure to replace with `revng_abort`.